platform/src/test/java/platformfeatures/PF_Membership.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Validate buttons on member plan page
+
+@MembershipPlan
+Scenario Outline: Validate all the buttons on member plan page
+
+Given Navigate to URL<browser> 
+Then Login To PF Portal 
```

Comment:
This can be Given or When step

Reply:
Yes I have reframed feature file as you have suggested.

Reply:
Done changes.